README.md

```diff
@@ -16,8 +16,8 @@ This repo contains .NET Core samples that demonstrate how to use the Azure Digit
 
 See the `README.md` in each sub-folder for specific details about each app:
 
-* [Occupancy sample](./occupancy-quickstart/README.md)
-* [Device Connectivity sample](./device-connectivity/README.md)
+* [Occupancy sample](https://github.com/Azure-Samples/digital-twins-samples-csharp/tree/master/occupancy-quickstart/README.md)
```

Comment:
As a general (github) usage - the downside of this is that the link from every branch will point to master's version.  So if you switch branch and view that and click the links it will take you to possibly out of sync content.  But maybe with the rest of the system in play this is a lesser of the problems.  Anyway up to you.

Reply:
I appreciate the feedback! 

Yes, that's a trade-off and is a definite cost in a multi-branch repo with many concurrent changes (and especially documentation ones) … hmm... I would hope given that it's been a bit quiet as of late and the number of documentation changes (in master and between branches) is fairly stable that it's OK to proceed this way. 

I also think the lack of 404's in Docs Samples is a major plus. Users today can click through to the README itself on GitHub first (resolving the issue) but they might begin excitedly clicking and this would be a decided improvement for those cases.

Reply:
Makes sense if Docs Samples doesn't handle these well as relative.  Thanks for the details.